### PR TITLE
rm is-stream

### PR DESCRIPTION
**Pull request by jimmywarting**
sprinkled little optional chaining into it

### CODE REVIEW

**Comment by ehmicky** [comment]
Hi @jimmywarting,

Thanks a lot for your contribution!

I do not have a strong opinion either for or against inlining `is-stream`. However, I believe @sindresorhus has had some concerns about such inlining PRs in the past, so I'd be curious to know what he thinks.

File: lib/stream.js [hunk 1]
```diff
@@ -81,7 +77,7 @@ export const getSpawnedResult = async ({stdout, stderr, all}, {encoding, buffer,
 };
 
 export const validateInputSync = ({input}) => {
-	if (isStream(input)) {
+	if (typeof input?.pipe === 'function') {
```
Comment (sindresorhus) [inline lib/stream.js:80]: For readability, make a `isStream` method.

File: lib/stream.js [hunk 2]
```diff
@@ -8,11 +7,8 @@ export const handleInput = (spawned, input) => {
 		return;
 	}
 
-	if (isStream(input)) {
-		input.pipe(spawned.stdin);
-	} else {
-		spawned.stdin.end(input);
-	}
+	// eslint-disable-next-line no-unused-expressions
+	input?.pipe?.(spawned.stdin) || spawned.stdin.end(input);
```
Comment (sindresorhus) [inline lib/stream.js:11]: This is not an improvement.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
I'm ok with removing it. The PR needs a better title though.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**